pkg/provider/workspace.go

```diff
@@ -204,6 +204,8 @@ type CLIOptions struct {
 	GitCloneStrategy     git.CloneStrategy `json:"gitCloneStrategy,omitempty"`
 	FallbackImage        string            `json:"fallbackImage,omitempty"`
 	GitSSHSigningKey     string            `json:"gitSshSigningKey,omitempty"`
+	GitOverrideUsername  string            `json:"gitUsername,omitempty"`
```

Comment:
If we have it on the CLIOptions already, do we still need it on this struct?

Reply:
currently we have it set on UpCmd struct and not workspace CLIOptions

Reply:
whoopsies, my bad! thanks